<launch>

      <node pkg="panda_gym_integration" 
            type="panda_path.py" 
            name="panda_path_node" 
            respawn="false" 
            output="screen">

            <!-- parameters-->
            <param name="mode" type="int" value="2"/>
            <param name="mode2_delay" type="double" value="1.0"/>
            <param name="file_name" type="string" value="path_e2e"/>
            <param name="real_robot" type="bool" value="false"/>

            <param name="gripper_speed" type="double" value="0.1"/>
            <param name="grasp_speed" type="double" value="0.1"/>
            <param name="grasp_force" type="double" value="20.0"/>
            <param name="grasp_epsilon_inner" type="double" value="0.02"/>
            <param name="grasp_epsilon_outer" type="double" value="0.02"/>

            <param name="eef_step" type="double" value="0.01"/>
            <param name="jump_threashould" type="double" value="0.0"/>
      </node>
      
</launch>